### PR TITLE
add posts about WildFly Docker images with Temurin

### PR DESCRIPTION
**Pull request by jmesnil**
Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

**Comment by jmesnil** [comment]
@bstansberry Could you please review? (I've checked locally that the article display fine).

When you are ok with the content, the plan is to:

* Merge https://github.com/jboss-dockerfiles/wildfly/pull/161 & https://github.com/jboss-dockerfiles/wildfly/pull/164 to be able to build the new images
* Build them for WildFly 26.1.2.Final (I'll remove the JDK 19 images that are automatically built as we did not test WildFly 26 with Temurin 19)
* Update the Dockerfile to build the WildFly 27.0.0.Final images
* Merge and publish this article

File: _posts/2022-11-11-wildfly-docker-temurin.adoc [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+Thanks to contributions from https://github.com/Henri-Laiho[Henri-Laiho] and https://github.com/krisgerhard[Kris-Gerhard], we have updated the images to use the JDK images from https://adoptium.net/[Eclipe Temurin] as their base images.
+This provides the following highly requested capabilities:
+
+* Provides multiarch images. The WildFly images now support `linux/arm64` in addition to `linux/amd64`
+* Supports Long-Term Support (LTS) JDK 11 & 17
+* Supports the latest non-LTS JDK (`19` at the moment)
+* Base images that are frequently updated to fix OS and JDK vulnerabilities
+
+We are using the `centos7` variants of the Temurin images, so there is no Operating System changes compared to the existing images.
```
Comment (bstansberry) [inline _posts/2022-11-11-wildfly-docker-temurin.adoc:20]: s/there is/there are/g

File: _posts/2022-11-11-wildfly-docker-temurin.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Docker Images for WildFly 27 and future releases
+
+Now that WildFly publishes images to run the application server with different JDK versions,we are updating our tag convention to reflect that change: _the tag of the image now identifies the version of WildFly as well as the JDK version_.
```
Comment (bstansberry) [inline _posts/2022-11-11-wildfly-docker-temurin.adoc:24]: s/versions,we/versions, we/g

**Review by bstansberry — APPROVED** [review]
Minor copy-editing tweaks aside, LGTM.

**Comment by bstansberry** [comment]
@jmesnil once this is merged I'll update the 'New stuff / Cloud' section of today's WF 27 release announcement post to add a mention of/link to this.

**Comment by jmesnil** [comment]
> Minor copy-editing tweaks aside, LGTM.

fixed, thanks

**Pull request merged**